### PR TITLE
add ticket link and update sessions page

### PR DESCRIPTION
**Pull request by refset**

### CODE REVIEW

File: conference/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 id: conference
 ---
 
+<script src='https://js.tito.io/v1' async></script>
```
Comment (rossjones) [inline conference/index.html:7]: Did you intend to load the https version? Does it complain?

Comment (refset) [inline conference/index.html:7]: Oops, I didn't mean to leave that there as it's not being used!

Comment (refset) [inline conference/index.html:7]: Can confirm it doesn't complain on my local environment. I can remove it if you'd prefer ...but my git-fu isn't great so I'll probably just create a new PR

**Comment by rossjones** [comment]
Have merged, could you check it over please @refset - ta

**Pull request merged**